content/en/blog/2022-07-05-unikraft-gsoc-intel-sgx.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Improving security and isolation of Unikraft with Intel SGX "
+date = "2022-06-13T00:00:00+01:00"
+authors = [
+  "Xiangyi Meng"
+]
+tags = ["Security", "Intel SGX"]
++++
```

Comment:
Please add the GSoC banner to the article as shown here: https://github.com/unikraft/docs/blob/main/content/en/blog/2022-05-21-unikraft-gsoc-org.md

```
<img width="100px" src="https://summerofcode.withgoogle.com/assets/media/gsoc-2022-badge.svg" align="right" />
```